at_client/src/main/java/org/atsign/client/util/RegisterUtil.java

```diff
@@ -0,0 +1,108 @@
+        httpsConnection.setRequestProperty("Authorization", Constants.DEV_API_KEY);
+        httpsConnection.setDoOutput(true);
+        OutputStream outputStream = httpsConnection.getOutputStream();
+        outputStream.write(params.getBytes(StandardCharsets.UTF_8), 0, params.length());
+        outputStream.flush();
+        outputStream.close();
+        System.out.println(httpsConnection.getResponseCode());
+        System.out.println(httpsConnection.getResponseMessage());
+        if (httpsConnection.getResponseCode() == HttpsURLConnection.HTTP_OK) {
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(
+                    httpsConnection.getInputStream()));
+            StringBuffer response = new StringBuffer();
+            response.append(bufferedReader.readLine());
+            if (response.toString().contains("Sent Successfully")) {
+                return true;
+            }
+            return false;
+        }
+        throw new AtException(httpsConnection.getResponseCode() + " " + httpsConnection.getResponseMessage());
+    }
+
+    //Accepts your email, unpaired atsign, and the otp received on the provided email.
+    //Validates the one-time-password against the atsign and registers it to the provided email if valid.
+    //Returns the CRAM secret pertaining to the atsign which is registered.
+    public String validateOtp(String email, String atsign, String otp) throws IOException, AtException {
+        String validationUrl = Constants.AT_DEV_DOMAIN + Constants.API_PATH + Constants.VALIDATE_OTP;
+        URL validateOtpUrl = new URL(validationUrl);
+        HttpsURLConnection httpsConnection = (HttpsURLConnection) validateOtpUrl.openConnection();
+        String params = "{\"atsign\":\"" + atsign + "\", \"email\":\"" + email + "\", \"otp\":\"" + otp
+                + "\", \"confirmation\":\"" + "true\"}";
+        System.out.println(params);
+        httpsConnection.setRequestMethod("POST");
+        httpsConnection.setRequestProperty("Content-Type", "application/json");
+        httpsConnection.setRequestProperty("Authorization", Constants.DEV_API_KEY);
+        httpsConnection.setDoOutput(true);
+        OutputStream outputStream = httpsConnection.getOutputStream();
+        outputStream.write(params.getBytes(StandardCharsets.UTF_8), 0, params.length());
+        outputStream.flush();
+        outputStream.close();
+        System.out.println(httpsConnection.getResponseCode());
+        System.out.println(httpsConnection.getResponseMessage());
+        if (httpsConnection.getResponseCode() == HttpsURLConnection.HTTP_OK) {
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(
+                    httpsConnection.getInputStream()));
+            StringBuffer response = new StringBuffer();
+            response.append(bufferedReader.readLine());
+            JSONObject dataJsonObject = new JSONObject(response.toString());
```

Comment:
If Java lang has support for serializing and deserializing from and to JSON it is better we use it so that we can reduce external dependencies. Generally, it is better to depend on things with the Java lang libraries.